timemachine/fe/absolute_hydration.py

```diff
@@ -242,8 +244,19 @@ def estimate_absolute_free_energy(
 
     combined_prefix = get_mol_name(mol) + "_" + prefix
     try:
-        return estimate_free_energy_given_initial_states(
-            initial_states, md_params, temperature, combined_prefix, keep_idxs
+        u_kln_by_component_by_lambda, stored_frames, stored_boxes = run_sequential_sims_given_initial_states(
```

Comment:
Decoupling simulation and analysis here should allow bisection to be almost a drop-in replacement